File: Java/Ram.java
public class Ram {
    private String capacity;    //menampung kapasitas dan harga
    private int price;          

    // constructor
    public Ram() {
    }

    public Ram(String capacity, int price) {
        this.capacity = capacity;
        this.price = price;
    }

    // setter and getter
    public void setCapacity(String capacity) {
        this.capacity = capacity;
    }

    public String getCapacity() {
        return this.capacity;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public int getPrice() {
        return this.price;
    }

    // method
    public void printRam() {
        System.out.println("Kapasitas RAM: " + this.getCapacity());
        System.out.println("Harga RAM : Rp " + this.getPrice());
    }
} 

File: Python/Ram.py
class Ram:
    #menampung kapasitas dan harga
    __capacity = "" 
    __price = 0     

    # constructor
    def __init__(self, capacity = "", price = 0):
        self.__capacity = capacity
        self.__price = price

    # setter and getter
    def setCapacity(self, capacity):
        self.__capacity = capacity

    def getCapacity(self):
        return self.__capacity
    
    def setPrice(self, price):
        self.__price = price
    
    def getPrice(self):
        return self.__price

    # method
    def printRAM(self):
        print("Kapasitas RAM : " + str(self.getCapacity()))
        print("Harga RAM : Rp " + str(self.getPrice()))